### PR TITLE
feat: allow forceNewDeployment

### PR DESCRIPTION
**Pull request by andhikamaheva**
Allow to force new deployment when updating service

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -487,7 +488,7 @@ describe('Deploy to ECS', () => {
 
         expect(mockEcsUpdateService).toHaveBeenCalledTimes(0);
         expect(mockEcsWaiter).toHaveBeenCalledTimes(0);
-    });
+    });
```
Comment (iamhopaul123) [inline index.test.js:491]: could you remove the trailing whitespace?

File: index.test.js [hunk 2]
```diff
@@ -497,6 +498,7 @@ describe('Deploy to ECS', () => {
             .mockReturnValueOnce('cluster-789')         // cluster
             .mockReturnValueOnce('false')               // wait-for-service-stability
             .mockReturnValueOnce('')                    // wait-for-minutes
+            .mockReturnValueOnce('false')               // wait-for-service-stability
```
Comment (iamhopaul123) [inline index.test.js:501]: I thought it is defaulted to be false. Or should we fix the comment?

**Review by iamhopaul123 — COMMENTED** [review]
Overall LGTM with just two nits! Thank you for contributing to this!

File: index.test.js (hunk 1 above)
Comment (andhikamaheva) [reply index.test.js]: Done, thank you

File: index.test.js (hunk 2 above)
Comment (andhikamaheva) [reply index.test.js]: Yes, default value is `false`. I've fixed the comment and test

**Review by iamhopaul123 — APPROVED** [review]
ship it :shipit:

**Comment by ltamrazov** [comment]
Hi, for clarification, is this meant to address #113 ?

**Comment by jleinext** [comment]
Any news on when this will be available? :)

**Comment by iamhopaul123** [comment]
@allisaurus, @piradeepk could you help me to take a look?

**Comment by mding5692** [comment]
Looking forward to this one, right now I'm handling it manually

**Review by allisaurus — APPROVED** [review]

**Pull request merged**